### PR TITLE
use harbor-sample-certificate

### PR DESCRIPTION
**Pull request by bitsf**
should use harbor-sample-certificate instead of public-certificate

### CODE REVIEW

File: config/samples/certificate.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ kind: Certificate
 metadata:
   name: harbor-sample-public
 spec:
-  secretName: public-certificate
+  secretName: harbor-sample-certificate
```
Comment (holyhope) [inline config/samples/certificate.yaml]: The goal of this Certificate resource is to generate the [`tlsSecretName` of harbor sample](https://github.com/goharbor/harbor-operator/blob/master/config/samples/goharbor_v1alpha1_harbor.yaml#L7).  
[./config/samples/goharbor_v1alpha1_harbor.yaml](https://github.com/goharbor/harbor-operator/blob/master/config/samples/goharbor_v1alpha1_harbor.yaml#L7) needs to be updated too.  
Such as other reference in documentation [./docs/installation.md](https://github.com/goharbor/harbor-operator/blob/master/docs/installation.md#deploy-the-sample)

**Comment by bitsf** [comment]
hi @holyhope 
I just wonder how harbor-sample-certificate and harbor-sample-notary-certificate created, and is harbor-sample-certificate designed for harbor nginx https ?

image

**Comment by holyhope** [comment]
These certificates are created by the controller and are for _internal_ use, it needs more documentation. I just created the issue #18.
Otherwise I suggest to rename `public-certificate` to `harbor-sample-public-certificate`, WDYT?

**Comment by bitsf** [comment]
Sounds good.
I thought public-certificate is used by mistake.
Ss harbor-sample-certificate is used between different components, such as core and jobservice, right?

**Comment by holyhope** [comment]
>  Ss harbor-sample-certificate is used between different components, such as core and jobservice, right?

That is it.

**Pull request closed without merging**